server/lib/notifications/email.ts

```diff
@@ -577,15 +577,16 @@ export const notifyByEmail = async (activity: Activity) => {
       break;
 
     case ActivityTypes.COLLECTIVE_VIRTUAL_CARD_SUSPENDED:
-    case ActivityTypes.COLLECTIVE_VIRTUAL_CARD_DELETED:
+    case ActivityTypes.COLLECTIVE_VIRTUAL_CARD_DELETED: {
+      activity.data.hostCollective = activity.data.host;
```

Comment:
I don't get this part; since host is already set in `activity.data.host`, why alias it and not use this variable directly?